super_calendar/README.rst

```diff
@@ -23,13 +23,15 @@ phone calls, you can create the following lines
 
 Meetings:
 
-.. image:: super_calendar/static/description/meetings.png
```

Comment:
Could you please update the README file to the newest template available here: https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst